components/home/homepage-whatsapp-cta.tsx

```diff
@@ -9,7 +9,7 @@ export function HomePageWhatsAppCTA() {
       <h2 className="font-semibold">Masih belum dapat info yang kamu cari?</h2>
       <PrimaryAnchorButton
         block
-        className="text-gray-800 bg-[#25d366] focus:ring-[#25d366]"
+        className="text-white bg-[#25d366] focus:ring-[#25d366]"
```

Comment:
To simulate how people with vision deficiencies see our website, we can use this Google Chrome's feature, as described in [this article](https://www.theverge.com/2020/3/11/21174735/google-chrome-dev-tools-new-color-blind-friendly) or [this tweet](https://twitter.com/mathias/status/1237393102635012101).

Here's an illustration of how people with vision deficiencies see our website before and after this Pull Request:

https://user-images.githubusercontent.com/6315466/129425979-707e3e10-7588-46a3-b67b-6f6c6ac2e307.mov

What do you think, Mas @mazipan, Mbak @ekafyi, Mas @isatrio?
FYI, @kawalcovid19/wbw.

Reply:
Basically we are not using WhatsApp API.
I suggest to not using WhatsApp Logo in the button.

JusJust use heroicons

![Screenshot_20210814-073555_Chrome](https://user-images.githubusercontent.com/7221389/129429269-70836b55-8381-46d4-856e-3498e65c006d.jpg)

Reply:
This project might not be using any WhatsApp API, but I believe the Hotline project (our WhatsApp business integration with Qiscus and Kata.ai) uses it. Therefore, I think it's still relevant to use the official WhatsApp logo in that button. 😅